src/liballoc/arc.rs

```diff
@@ -251,7 +253,9 @@ impl<T: ?Sized> Arc<T> {
             let cur = this.inner().weak.load(Relaxed);
 
             // check if the weak counter is currently "locked"; if so, spin.
-            if cur == usize::MAX { continue }
+            if cur == usize::MAX {
+                continue
```

Comment:
since this is a style PR, I wonder: is it good style to use `;` here? I probably would. I know it's not in the original source, so it's not `rustfmt` doing it.